tests/fixtures/manifests/simple.wadm.yaml

```
@@ -9,20 +9,21 @@ metadata:
 spec:
   policies:
     - name: vault-for-component
-      type: secrets.wasmcloud.dev/v1/vault
+      type: policy.secret.wasmcloud.dev/v1alpha1
       properties:
-        role-name: "test"
-        backend: "vault-prod"
+        role-name: 'test'
+        backend: 'vault-prod'
```

If we want to stay true to what's implemented, this would be a good representation:
```suggestion
        backend: 'vault-prod'
        role_name: 'test'
        mount_path: 'jwt'
```